requirements.txt

```diff
@@ -2,7 +2,7 @@ future
 zsl_client
 click>=6.6
 Flask>=0.11
-Flask-Injector==0.8
+Flask-Injector>=0.8
```

Comment:
The version must be set to ==0.8, with 0.9 they went full py3 and broke 2.7 compatibility. The added type annotation to go with the newer python injector DI style.